config/typescript/modern.json

```diff
@@ -0,0 +1,38 @@
+{
+	"compilerOptions": {
```

Comment:
@InteractiveTimmy  how does output directory be set, in older tsconfig I see 

```
    "outDir": "./dist/module",
    "declarationDir": "./dist/types",
    ```

Reply:
these are handled within `./package.json`'s scripts, or can be handled within `./tsconfig.json` for any particular package. By placing this within the `scripts` of the `./package.json`, we can easily set up a workflow for types and modules without needing two tsconfig files.